react/features/base/participants/components/Avatar.native.js

```diff
@@ -229,13 +205,13 @@ export default class Avatar extends Component<Props, State> {
         // regular Image, so we need to wrap it in a view to make it round.
         // https://github.com/facebook/react-native/issues/3198
 
-        const { backgroundColor, useDefaultAvatar } = this.state;
+        const { backgroundColor } = this.state;
         const imageStyle = this._getImageStyle();
         const viewStyle = {
             ...imageStyle,
 
             backgroundColor,
-            display: useDefaultAvatar ? 'flex' : 'none',
+            display: this.useDefaultAvatar ? 'flex' : 'none',
```

Comment:
Why is there a check for the truthiness of "useDefaultAvatar" in the method renderDefaultAvatar? Should it not always be true?